cdap-data-fabric/src/main/java/co/cask/cdap/data2/datafabric/dataset/service/DatasetInstanceService.java

```diff
@@ -488,12 +489,10 @@ private void enableExplore(DatasetId datasetInstance, DatasetSpecification spec,
     try {
       exploreFacade.enableExploreDataset(datasetInstance.toId(), spec);
     } catch (Exception e) {
-      String msg = String.format("Cannot enable exploration of dataset instance %s of type %s: %s",
-                                 datasetInstance, creationProperties.getProperties(), e.getMessage());
-      LOG.error(msg, e);
+      LOG.error("Cannot enable explore for dataset instance {} of type {} with properties {}: {}",
```

Comment:
`e.getMessage` seems redundant, since we're passing `e` as the last parameter.
Won't the message get logged twice now?